WorkoutTest/ProgramViewController.swift

```diff
@@ -92,6 +92,16 @@ extension ProgramViewController: UITableViewDelegate {
     let selectedProgram = welcome?.programs[indexPath.row]
     let vc = DayViewController()
     vc.program = selectedProgram
+    vc.confirmDelete = { [weak self] in
+      self?.deleteProgram(atIndex: indexPath.row)
+    }
     navigationController?.pushViewController(vc, animated: true)
   }
+  // MARK: - Function for deleting a cell
+  
+  func deleteProgram(atIndex index: Int) {
```

Comment:
consider using this function in  `func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration?` to avoid code duplication